cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/README.md

```diff
@@ -0,0 +1,32 @@
+# Package Name
+oradb_discovery_v3_6_11_19.sql
+
+## Description
+The Database discovery & analysis tool is a SQL utility that captures necessary details of an oracle database in order to analyse on-Prem or cloud Oracle Databases for the suitability of migration/upgrades. This utility runs on Non Container or Container databases and helps in generating HTML report with detailed description
```

Comment:
Change '&' to 'and'.
Change 'on-Prem' to 'on-premises'.
Always end a sentence properly, e.g., "in generating HTML report with detailed description."